package.json

```diff
@@ -4,16 +4,21 @@
   "description": "Build/Layout optimization tooling and Peregrine framework adapters for the Magento PWA",
   "main": "dist/index.js",
   "scripts": {
-    "build": "babel src --out-dir dist --ignore __tests__ --source-maps --copy-files",
-    "lint": "eslint '*.js' 'src/**/*.js'",
     "test": "run-p jest:ci prettier:check lint",
-    "test:dev": "jest --watch",
-    "jest:ci": "jest -i --forceExit --testResultsProcessor=./node_modules/jest-junit-reporter",
+    "test:dev": "run-s prettier lint jest:ci",
+    "test:watch": "jest --watch --runInBand",
```

Comment:
`--runInBand` is going to slow down watching in development - doubt that was the intention here?